boards/nxp/frdm_mcxa156/frdm_mcxa156.yaml

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: frdm_mcxa156
+name: NXP FRDM MCXA156
+type: mcu
+arch: arm
+ram: 132
```

Comment:
Does the part have 132KB of ram, or 128KB as the documentation states?

Reply:
Thanks for reminder, update both to 128KB.